### PR TITLE
Channel/octave in MTS-ESP mode

### PR DESCRIPTION
**Pull request by Andreya-Autumn**
- Allow channel per octave mode under MTS-ESP
- Show menu item when MTS-ESP is active
- Re-structure MTS-ESP tuning menu somewhat

### CODE REVIEW

File: src/common/dsp/SurgeVoice.cpp [hunk 1]
```diff
@@ -215,8 +216,10 @@ SurgeVoice::SurgeVoice(SurgeStorage *storage, SurgeSceneStorage *oscene, pdata *
     const bool isCh23Mode = Surge::Storage::getUserDefaultValue(
         storage, Surge::Storage::UseCh2Ch3ToPlayScenesIndividually, true, false);
     const bool isChSplitMode = storage->getPatch().scenemode.val.i == sm_chsplit;
+    const bool ChannelPerOctave = storage->mapChannelToOctave;
 
-    state.mtsUseChannelWhenRetuning = (mpeEnabled || isChSplitMode || isCh23Mode);
+    state.mtsUseChannelWhenRetuning =
+        (mpeEnabled || isChSplitMode || isCh23Mode || ChannelPerOctave);
```
Comment (mkruselj) [inline src/common/dsp/SurgeVoice.cpp:222]: You could just directly use storage->mapChannelToOctave here, no need to have it as a variable when it's only used once.

Comment (Andreya-Autumn) [reply src/common/dsp/SurgeVoice.cpp:222]: Right, I thought I'd match what the other parts of the check were doing. But that makes sense. Will change.

**Comment by Andreya-Autumn** [comment]
This all works for me and checks passed. Please take a look in case I did something silly though. ;) As time permits of course.

**Comment by baconpaul** [comment]
Yup

**Pull request merged**